ops/testing.py

```diff
@@ -1518,14 +1518,63 @@ def add_layer(
                 if service.override not in ('merge', 'replace'):
```

Comment:
+1 if you make that tuple a set() for faster lookup

Reply:
Meh - for ~several entries, performance will be identical.